livre/core/Render/GLContext.h

```diff
@@ -43,6 +43,8 @@ class GLContext : public boost::enable_shared_from_this< GLContext >
      */
     void shareContext( GLContextPtr dstContextPtr );
 
+    const GLEWContext* glewGetContext() const;
```

Comment:
GLEWContext != GLContext

The glew context is the OpenGL function table to call GL functions. On windows most gl functions are not defined as symbols (ie. glCreateShader), and are only available as function pointers from wglGetProcAddress( "glCreateShader" ). Any other OS does this in a more reasonable way, guess why.

The GL context sharing is a different problem.